server/game/costs.js

```diff
@@ -210,6 +210,19 @@ const Costs = {
         };
     },
     /**
+     * Cost that will put into play the card that initiated the ability.
+     */
+    putSelfIntoPlay: function() {
+        return {
+            canPay: function() {
+                return true;
```

Comment:
You should check `canPutIntoPlay` to handle cases where the character may be in the dead pile / you've take control of a copy of the same card from an opponent / etc.